lib/migrate/uglifyJsPlugin/__snapshots__/uglifyJsPlugin.test.js.snap

```diff
@@ -2,36 +2,124 @@
 
 exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-0" data 1`] = `
 "module.exports = {
-    plugins: [
-        new webpack.optimize.UglifyJsPlugin({
-            sourceMap: true
-        })
-    ]
+    plugins: [],
```

Comment:
Why's this empty? Should be preserved when `uglifyJSPlugin` is correct?

Reply:
I missed this cleanup. I'm going to create an utility to remove plugins property if they are empty, so it can be used by other transformations that fell into the same problem.